kernel/sched.c

```diff
@@ -975,9 +975,9 @@ s32_t z_impl_k_sleep(int ms)
 {
 	s32_t ticks;
 
-	ticks = z_ms_to_ticks(ms);
+	ticks = k_ms_to_ticks_ceil32(ms);
```

Comment:
i would add `if(ms <= 0) return 0` to handle negative input (which may come from arithmetic operation result).

Reply:
This is just an automated change to pre-existing code.  That's an unrelated fix, no?